src/sxastarter/src/lib/dictionary-service-factory.ts

```diff
@@ -13,7 +13,6 @@ export class DictionaryServiceFactory {
           endpoint: config.graphQLEndpoint,
           apiKey: config.sitecoreApiKey,
           siteName: config.jssAppName,
-          jssAppTemplateId: '{9ED66404-64C9-4122-90E1-869CB3CEA566}',
```

Comment:
Please revert this. This is essential to solve "rootItemId is not resolved" errors when fetching dictionary. (the SXA site root template is different than vanilla JSS site root)